src/styles/blocks/review.css

```diff
@@ -0,0 +1,34 @@
+.review {
+  align-self: flex-start;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 16px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  display: inline-block;
+  margin: 2px 6px 2px 2px;
+  height: 12px;
+  width: 12px;
+  background-image: url(../../images/star.svg);
+  background-size: cover;
+}
+
+.review__stars :nth-child(-n+4) {
+  background-image: url(../../images/star-active.svg);
+}
```

Comment:
You need to be able to set active stars with a modifier in HTML. This CSS code doesn't seem to support that.

Reply:
Done